src/transformers/models/codegen/configuration_codegen.py

```diff
@@ -57,6 +57,7 @@ class CodeGenConfig(PretrainedConfig):
         n_positions (`int`, *optional*, defaults to 2048):
             The maximum sequence length that this model might ever be used with. Typically set this to something large
             just in case (e.g., 512 or 1024 or 2048).
+        n_ctx (`int`, *optional*, defaults to 2048): Number of tokens provided in a single pass.
```

Comment:
Let's make the description (like `Number of tokens provided in a single pass.`) be in a separate line with indent.

Same comments applied to other places.